scripts/propagate_doc_upwards.bash

```diff
@@ -48,7 +48,6 @@ log "Configuration: REPO_NAME=${REPO_NAME}"
 git config merge.ours.driver true
 
 if [ "${REPO_NAME}" == "bonita-doc" ]; then
-  merge "2022.1" "2022.2"
```

Comment:
**nitpick:** out of the scope of the 7.10 removal, must be done in a dedicated PR